src/scripts/lib/data.json

```diff
@@ -0,0 +1,1302 @@
+[
```

Comment:
@hasnaindev Should this be categorised as `lib`?

Reply:
Hmm, any suggestions?

Reply:
I am not sure. Maybe put the `data.json` right in the `scripts` directory. Or better inspect other open source projects and see what they are doing.

For instance, this project is organising the JSON data in `assets/data/countries/{country}.json` but since we have a single data file, it'll make sense to put it directly in `src` or at most `src/scripts` but it's not a script though.